hpi-swt2/lecture-portal
# Feature/#38 ended lectures

# Resolves #38 

This PR changes the index lecture view, so one can't restart an ended lecture

Acceptance Criteria:
- [x] can't restart an ended lecture 

Testing steps:
- Sign in as a lecturer
- Start a lecture
- End the lecture
- No start button for the ended lecture should be there

Steps needed for migration:
- first migration step
- second migration step
- ...

---

Definition Of Done:
